Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -308,7 +308,7 @@ extension ProjectDescription.Target {
         )
 
         return ProjectDescription.Target(
-            name: target.name,
+            name: target.name.replacingOccurrences(of: ".", with: "_"),
```

Comment:
What about defining a static function on `PackageInfoMapper` so you can invoke it here and below without repeating the logic?
Something like:
```
            name: PackageInfoMapper.sanitize(targetName: target.name)),
```

Reply:
I like it, will do 👍